gqlgen.yml

```diff
@@ -48,10 +48,6 @@ models:
     model: github.com/evergreen-ci/evergreen/rest/model.APIClientConfig
   CloudProviderConfig:
     model: github.com/evergreen-ci/evergreen/rest/model.APICloudProviders
-  CommitQueue:
-    model: github.com/evergreen-ci/evergreen/rest/model.APICommitQueue
-  CommitQueueItem:
-    model: github.com/evergreen-ci/evergreen/rest/model.APICommitQueueItem
   CommitQueueParams:
```

Comment:
Note: these fields weren't deleted because they're used in Project and Repo Refs.